test/DynamoCoreWpf3Tests/UnitsUITests.cs

```diff
@@ -22,7 +22,6 @@ public class UnitsUITests : DynamoTestUIBase
         private MeasurementInputBaseConcrete measurementInputBase;
         private AssemblyHelper assemblyHelper;
 
-
```

Comment:
The test cases were updated to compare only type names (e.g., `"autodesk.unit.quantity:force"`) instead of full type IDs (e.g., `"autodesk.unit.quantity:force-1.0.2"`), making the tests less sensitive to schema version changes.